src/Api/Management/Entity/App.php

```diff
@@ -216,4 +219,20 @@ public function setScopes(string ...$scopes): void
     {
         $this->privateSetScopes(...$scopes);
     }
+
+    /**
+     * @param array $apiProducts
+     */
+    public function getApiProducts(): array
+    {
+        return $this->initialApiProducts;
```

Comment:
Returning an empty array here would not be misleading? Since API Product association is stored on credentials and not on the app.

Reply:
Agreed. With `getApiProducts` it will post to Edge and `apiProducts` is an IN parameter but in the return we will get inside the credential so there is no `setApiProducts`. Only for App creation will have 'apiProducts', not for the update